client/src/components/Admin/Jobs/index.vue

```diff
@@ -94,9 +101,78 @@ export default {
             loading: false,
             jobList: [],
             root: process.env.VUE_APP_ROOT_API,
+            createJobModal: false,
+            jobModal: false,
         }
     },
+    mounted() { this.jobDetail, this.selectedStatus },
     methods: {
+        jobDetail(val) {
+            const isEmptyObjectValuesEmpty = Object.values(val).every(value => {
+                return value === '' || value === null || value === undefined;
+            });
+            if (isEmptyObjectValuesEmpty == true) {
+                this.$bvToast.toast("Please fill all the details", {
+                    title: "Error",
+                    variant: "danger",
+                    solid: true,
+                });
+            }
+            else {
+                Object.assign(val, { adminId: localStorage.getItem("adminId") });
```

Comment:
why are you assigning `adminId` at runtime? 
add as vdata to `jobComponent` and add it to jobObj there.. simple?

Reply:
Inorder to make the object empty for validation and Now I have changed it to the component